diff --git a/src/hooks/useScrollEventsHandlersDefault.ts b/src/hooks/useScrollEventsHandlersDefault.ts
index c0594494415a0f996aafce7a67e7175ee26117d8..27f1c49191c094161ba96269f6cb526dd6955deb 100644
--- a/src/hooks/useScrollEventsHandlersDefault.ts
+++ b/src/hooks/useScrollEventsHandlersDefault.ts
@@ -55,6 +55,9 @@ export const useScrollEventsHandlersDefault: ScrollEventsHandlersHookType = (
           const lockPosition = context.shouldLockInitialPosition
             ? (context.initialContentOffsetY ?? 0)
             : 0;
+          if (y === lockPosition) {
+            return;
+          }
           // @ts-ignore
           scrollTo(scrollableRef, 0, lockPosition, false);
           scrollableContentOffsetY.value = lockPosition;
@@ -104,6 +107,9 @@ export const useScrollEventsHandlersDefault: ScrollEventsHandlersHookType = (
           const lockPosition = context.shouldLockInitialPosition
             ? (context.initialContentOffsetY ?? 0)
             : 0;
+          if (y === lockPosition) {
+            return;
+          }
           // @ts-ignore
           scrollTo(scrollableRef, 0, lockPosition, false);
           scrollableContentOffsetY.value = lockPosition;
@@ -134,9 +140,12 @@ export const useScrollEventsHandlersDefault: ScrollEventsHandlersHookType = (
           const lockPosition = context.shouldLockInitialPosition
             ? (context.initialContentOffsetY ?? 0)
             : 0;
+          if (y === lockPosition) {
+            return;
+          }
           // @ts-ignore
           scrollTo(scrollableRef, 0, lockPosition, false);
-          scrollableContentOffsetY.value = 0;
+          scrollableContentOffsetY.value = lockPosition;
           return;
         }
 
