diff --git a/src/hooks/useScrollEventsHandlersDefault.ts b/src/hooks/useScrollEventsHandlersDefault.ts
index 7b84480e8b002f843d349d4f47d4b4f9f66c17bb..62cfd61a9b29cad509bb4689f339d7e4f14a5b71 100644
--- a/src/hooks/useScrollEventsHandlersDefault.ts
+++ b/src/hooks/useScrollEventsHandlersDefault.ts
@@ -55,6 +55,9 @@ export const useScrollEventsHandlersDefault: ScrollEventsHandlersHookType = (
           const lockPosition = context.shouldLockInitialPosition
             ? (context.initialContentOffsetY ?? 0)
             : 0;
+          if (Math.abs(y - lockPosition) < 1) {
+            return;
+          }
           // @ts-expect-error
           scrollTo(scrollableRef, 0, lockPosition, false);
           scrollableContentOffsetY.value = lockPosition;
@@ -104,6 +107,9 @@ export const useScrollEventsHandlersDefault: ScrollEventsHandlersHookType = (
           const lockPosition = context.shouldLockInitialPosition
             ? (context.initialContentOffsetY ?? 0)
             : 0;
+          if (Math.abs(y - lockPosition) < 1) {
+            return;
+          }
           // @ts-expect-error
           scrollTo(scrollableRef, 0, lockPosition, false);
           scrollableContentOffsetY.value = lockPosition;
@@ -134,9 +140,12 @@ export const useScrollEventsHandlersDefault: ScrollEventsHandlersHookType = (
           const lockPosition = context.shouldLockInitialPosition
             ? (context.initialContentOffsetY ?? 0)
             : 0;
+          if (Math.abs(y - lockPosition) < 1) {
+            return;
+          }
           // @ts-expect-error
           scrollTo(scrollableRef, 0, lockPosition, false);
-          scrollableContentOffsetY.value = 0;
+          scrollableContentOffsetY.value = lockPosition;
           return;
         }
 
