diff --git a/lib/module/views/BottomTabBar.js b/lib/module/views/BottomTabBar.js
index cb4e310644f3bb2ec2a6a4358930dbfe620a9ce1..ecacb80eb56b474e62a7d66b19a952691803b55c 100644
--- a/lib/module/views/BottomTabBar.js
+++ b/lib/module/views/BottomTabBar.js
@@ -123,6 +123,7 @@ export function BottomTabBar({
     tabBarShowLabel,
     tabBarLabelPosition,
     tabBarHideOnKeyboard = false,
+    tabBarVisible = false,
     tabBarVisibilityAnimationConfig,
     tabBarVariant = 'uikit',
     tabBarStyle,
@@ -140,7 +141,7 @@ export function BottomTabBar({
   }
   const isKeyboardShown = useIsKeyboardShown();
   const onHeightChange = React.useContext(BottomTabBarHeightCallbackContext);
-  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown);
+  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown) && !tabBarVisible;
   const visibilityAnimationConfigRef = React.useRef(tabBarVisibilityAnimationConfig);
   React.useEffect(() => {
     visibilityAnimationConfigRef.current = tabBarVisibilityAnimationConfig;
@@ -241,8 +242,12 @@ export function BottomTabBar({
         translateY: visible.interpolate({
           inputRange: [0, 1],
           outputRange: [layout.height + insets[tabBarPosition === 'top' ? 'top' : 'bottom'] + StyleSheet.hairlineWidth, 0]
-        })
+        }),
       }],
+      opacity: visible.interpolate({
+        inputRange: [0, 1],
+        outputRange: [0, 1]
+     }),
       // Absolutely position the tab bar so that the content is below it
       // This is needed to avoid gap at bottom when the tab bar is hidden
       position: isTabBarHidden ? 'absolute' : undefined
diff --git a/src/types.tsx b/src/types.tsx
index 2438b03e744a2845fa6f7035586f64d4cc8e91c1..eaa6d4bcbf574075317d131c7a0b0b21b89d542c 100644
--- a/src/types.tsx
+++ b/src/types.tsx
@@ -239,6 +239,11 @@ export type BottomTabNavigationOptions = HeaderOptions & {
    */
   tabBarVariant?: Variant;
 
+  /**
+   * Whether the tab bar gets hidden when the screen is focused. Defaults to `false`.
+   */
+  tabBarVisible?: boolean;
+
   /**
    * Style object for the tab bar container.
    */
diff --git a/src/views/BottomTabBar.tsx b/src/views/BottomTabBar.tsx
index 55b9c8f3065ca5e887930af4ea9779838bb136cb..816b02840afb5a1007d79cfa4b740d063db348f8 100644
--- a/src/views/BottomTabBar.tsx
+++ b/src/views/BottomTabBar.tsx
@@ -175,6 +175,7 @@ export function BottomTabBar({
     tabBarVariant = 'uikit',
     tabBarStyle,
     tabBarBackground,
+    tabBarVisible = false,
     tabBarActiveTintColor,
     tabBarInactiveTintColor,
     tabBarActiveBackgroundColor,
@@ -205,7 +206,7 @@ export function BottomTabBar({
 
   const onHeightChange = React.useContext(BottomTabBarHeightCallbackContext);
 
-  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown);
+  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown) && !tabBarVisible
 
   const visibilityAnimationConfigRef = React.useRef(
     tabBarVisibilityAnimationConfig
@@ -367,6 +368,10 @@ export function BottomTabBar({
                         0,
                       ],
                     }),
+                    opacity: visible.interpolate({
+                      inputRange: [0, 1],
+                      outputRange: [0, 1],
+                    }),
                   },
                 ],
                 // Absolutely position the tab bar so that the content is below it
