diff --git a/node_modules/@react-navigation/bottom-tabs/lib/commonjs/views/BottomTabBar.js b/node_modules/@react-navigation/bottom-tabs/lib/commonjs/views/BottomTabBar.js
index 8f0fb1c..bfc1cfc 100644
--- a/node_modules/@react-navigation/bottom-tabs/lib/commonjs/views/BottomTabBar.js
+++ b/node_modules/@react-navigation/bottom-tabs/lib/commonjs/views/BottomTabBar.js
@@ -131,6 +131,7 @@ function BottomTabBar({
     tabBarShowLabel,
     tabBarLabelPosition,
     tabBarHideOnKeyboard = false,
+    tabBarVisible = false,
     tabBarVisibilityAnimationConfig,
     tabBarVariant = 'uikit',
     tabBarStyle,
@@ -149,7 +150,7 @@ function BottomTabBar({
   const dimensions = (0, _reactNativeSafeAreaContext.useSafeAreaFrame)();
   const isKeyboardShown = (0, _useIsKeyboardShown.useIsKeyboardShown)();
   const onHeightChange = _react.default.useContext(_BottomTabBarHeightCallbackContext.BottomTabBarHeightCallbackContext);
-  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown);
+  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown) && !tabBarVisible;
   const visibilityAnimationConfigRef = _react.default.useRef(tabBarVisibilityAnimationConfig);
   _react.default.useEffect(() => {
     visibilityAnimationConfigRef.current = tabBarVisibilityAnimationConfig;
@@ -251,6 +252,10 @@ function BottomTabBar({
           outputRange: [layout.height + insets[tabBarPosition === 'top' ? 'top' : 'bottom'] + _reactNative.StyleSheet.hairlineWidth, 0]
         })
       }],
+      opacity: visible.interpolate({
+        inputRange: [0, 1],
+        outputRange: [0, 1]
+      }),
       // Absolutely position the tab bar so that the content is below it
       // This is needed to avoid gap at bottom when the tab bar is hidden
       position: isTabBarHidden ? 'absolute' : undefined
diff --git a/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js b/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js
index dbde384..ec63cde 100644
--- a/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js
+++ b/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js
@@ -124,6 +124,7 @@ export function BottomTabBar({
     tabBarShowLabel,
     tabBarLabelPosition,
     tabBarHideOnKeyboard = false,
+    tabBarVisible = false,
     tabBarVisibilityAnimationConfig,
     tabBarVariant = 'uikit',
     tabBarStyle,
@@ -142,7 +143,7 @@ export function BottomTabBar({
   const dimensions = useSafeAreaFrame();
   const isKeyboardShown = useIsKeyboardShown();
   const onHeightChange = React.useContext(BottomTabBarHeightCallbackContext);
-  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown);
+  const shouldShowTabBar = !(tabBarHideOnKeyboard && isKeyboardShown) && !tabBarVisible;
   const visibilityAnimationConfigRef = React.useRef(tabBarVisibilityAnimationConfig);
   React.useEffect(() => {
     visibilityAnimationConfigRef.current = tabBarVisibilityAnimationConfig;
@@ -244,6 +245,10 @@ export function BottomTabBar({
           outputRange: [layout.height + insets[tabBarPosition === 'top' ? 'top' : 'bottom'] + StyleSheet.hairlineWidth, 0]
         })
       }],
+      opacity: visible.interpolate({
+        inputRange: [0, 1],
+        outputRange: [0, 1]
+      }),
       // Absolutely position the tab bar so that the content is below it
       // This is needed to avoid gap at bottom when the tab bar is hidden
       position: isTabBarHidden ? 'absolute' : undefined
diff --git a/node_modules/@react-navigation/bottom-tabs/src/types.tsx b/node_modules/@react-navigation/bottom-tabs/src/types.tsx
index 02f7119..01dfabc 100644
--- a/node_modules/@react-navigation/bottom-tabs/src/types.tsx
+++ b/node_modules/@react-navigation/bottom-tabs/src/types.tsx
@@ -224,6 +224,11 @@ export type BottomTabNavigationOptions = HeaderOptions & {
    */
   tabBarHideOnKeyboard?: boolean;
 
+  /**
+   * Whether the tab bar gets hidden when the screen is focused. Defaults to `false`.
+   */
+  tabBarVisible?: boolean;
+
   /**
    * Animation config for showing and hiding the tab bar when the keyboard is shown/hidden.
    */
