diff --git a/node_modules/@react-native-mapbox-gl/maps/.DS_Store b/node_modules/@react-native-mapbox-gl/maps/.DS_Store
new file mode 100644
index 0000000..a770bde
Binary files /dev/null and b/node_modules/@react-native-mapbox-gl/maps/.DS_Store differ
diff --git a/node_modules/@react-native-mapbox-gl/maps/.github/.DS_Store b/node_modules/@react-native-mapbox-gl/maps/.github/.DS_Store
new file mode 100644
index 0000000..5fb8787
Binary files /dev/null and b/node_modules/@react-native-mapbox-gl/maps/.github/.DS_Store differ
diff --git a/node_modules/@react-native-mapbox-gl/maps/index.d.ts b/node_modules/@react-native-mapbox-gl/maps/index.d.ts
index 10aa209..84ea37d 100644
--- a/node_modules/@react-native-mapbox-gl/maps/index.d.ts
+++ b/node_modules/@react-native-mapbox-gl/maps/index.d.ts
@@ -164,8 +164,8 @@ declare namespace MapboxGL {
 
    class AnimatedPoint {
     constructor(point?: GeoJSON.Point);
-    longitude: ReactNative.Animated.Value<number>;
-    latitude: ReactNative.Animated.Value<number>;
+    longitude: ReactNative.Animated.Value;
+    latitude: ReactNative.Animated.Value;
     setValue: (point: GeoJSON.Point) => void;
     setOffset: (point: GeoJSON.Point) => void;
     flattenOffset: () => void;
diff --git a/node_modules/@react-native-mapbox-gl/maps/javascript/.DS_Store b/node_modules/@react-native-mapbox-gl/maps/javascript/.DS_Store
new file mode 100644
index 0000000..5b6427a
Binary files /dev/null and b/node_modules/@react-native-mapbox-gl/maps/javascript/.DS_Store differ
diff --git a/node_modules/@react-native-mapbox-gl/maps/scripts/.DS_Store b/node_modules/@react-native-mapbox-gl/maps/scripts/.DS_Store
new file mode 100644
index 0000000..4e29e0c
Binary files /dev/null and b/node_modules/@react-native-mapbox-gl/maps/scripts/.DS_Store differ
