diff --git a/node_modules/@react-native-community/picker/typings/Picker.d.ts b/node_modules/@react-native-community/picker/typings/Picker.d.ts
index 6864f1f..44e7f51 100644
--- a/node_modules/@react-native-community/picker/typings/Picker.d.ts
+++ b/node_modules/@react-native-community/picker/typings/Picker.d.ts
@@ -1,7 +1,7 @@
 import * as React from "react"
 import { TextStyle, StyleProp, ViewProps } from 'react-native'
 
-type ItemValue  = number | string
+export type ItemValue  = number | string
 
 export interface PickerItemProps {
 	label?: string;
@@ -66,4 +66,4 @@ declare class Picker extends React.Component<PickerProps, {}> {
    static Item: React.ComponentType<PickerItemProps>;
 }
 
-export const Picker;
\ No newline at end of file
+export {Picker};
diff --git a/node_modules/@react-native-community/picker/typings/PickerIOS.d.ts b/node_modules/@react-native-community/picker/typings/PickerIOS.d.ts
index ef6745a..41f4721 100644
--- a/node_modules/@react-native-community/picker/typings/PickerIOS.d.ts
+++ b/node_modules/@react-native-community/picker/typings/PickerIOS.d.ts
@@ -9,7 +9,7 @@ export interface PickerIOSItemProps {
 	testID?: string;
 }
 
-class PickerIOSItem extends React.Component<PickerIOSItemProps, {}> {}
+declare class PickerIOSItem extends React.Component<PickerIOSItemProps, {}> {}
 
 export interface PickerIOSProps extends ViewProps {
 	itemStyle?: StyleProp<TextStyle>;
@@ -21,7 +21,7 @@ export interface PickerIOSProps extends ViewProps {
 }
 
 declare class PickerIOS extends React.Component<PickerIOSProps, {}> {
-	static Item: PickerIOSItem
+	static Item: typeof PickerIOSItem
 }
 
-export const PickerIOS;
\ No newline at end of file
+export { PickerIOS };
