diff --git a/node_modules/@segment/analytics-react-native-appboy/RNAnalyticsIntegration-Appboy.podspec b/node_modules/@segment/analytics-react-native-appboy/RNAnalyticsIntegration-Appboy.podspec
deleted file mode 100644
index a9eeb16..0000000
--- a/node_modules/@segment/analytics-react-native-appboy/RNAnalyticsIntegration-Appboy.podspec
+++ /dev/null
@@ -1,29 +0,0 @@
-require 'json'
-package = JSON.parse(File.read('./package.json'))
-
-Pod::Spec.new do |s|
-  s.name                = "RNAnalyticsIntegration-Appboy"
-  s.version             = package["version"]
-  s.summary             = package["description"]
-  s.description         = <<-DESC
-                          Analytics for React-Native provides a single API that lets you
-                          integrate with over 100s of tools.
-
-                          This is the Appboy integration for the React-Native library.
-                          DESC
-
-  s.homepage            = "http://segment.com/"
-  s.license             =  { :type => 'MIT' }
-  s.author              = { "Segment" => "friends@segment.com" }
-  s.source              = { :git => "https://github.com/segmentio/analytics-react-native.git", :tag => s.version.to_s }
-  s.social_media_url    = 'https://twitter.com/segment'
-
-  s.platform            = :ios, "11.0"
-  s.source_files        = 'ios/main.m'
-  s.static_framework    = true
-
-  s.dependency          'Segment-Appboy'
-  s.dependency          'Analytics'
-  s.dependency          'React-Core'
-  s.dependency          'RNAnalytics'
-end
