diff --git a/node_modules/@segment/analytics-react-native-appboy/android/build.gradle b/node_modules/@segment/analytics-react-native-appboy/android/build.gradle
index f3417bc..3d5157e 100644
--- a/node_modules/@segment/analytics-react-native-appboy/android/build.gradle
+++ b/node_modules/@segment/analytics-react-native-appboy/android/build.gradle
@@ -2,13 +2,14 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 buildscript {
+    ext.defaultKotlinVersion = '1.4.0'
     repositories {
         jcenter()
         maven { url 'https://maven.google.com' }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.4'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.21'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : defaultKotlinVersion}"
     }
 }
 
