diff --git a/node_modules/@sentry/react-native/dist/js/options.d.ts b/node_modules/@sentry/react-native/dist/js/options.d.ts
index 1996991..f26ec95 100644
--- a/node_modules/@sentry/react-native/dist/js/options.d.ts
+++ b/node_modules/@sentry/react-native/dist/js/options.d.ts
@@ -40,6 +40,10 @@ export interface ReactNativeOptions extends BrowserOptions {
     sessionTrackingIntervalMillis?: number;
     /** Enable scope sync from Java to NDK on Android */
     enableNdkScopeSync?: boolean;
+    // This is broken in Sentry and this property is not yet exposed although it's working properly
+    // See https://github.com/getsentry/sentry-react-native/issues/1633
+    /** Enable out of memory tracking */
+    enableOutOfMemoryTracking?: boolean;
     /** When enabled, all the threads are automatically attached to all logged events on Android */
     attachThreads?: boolean;
     /**
