diff --git a/node_modules/@storybook/client-api/dist/story_store.d.ts b/node_modules/@storybook/client-api/dist/story_store.d.ts
index d90acf1..4dd326e 100644
--- a/node_modules/@storybook/client-api/dist/story_store.d.ts
+++ b/node_modules/@storybook/client-api/dist/story_store.d.ts
@@ -14,6 +14,7 @@ import EventEmitter from 'eventemitter3';
 import { Channel } from '@storybook/channels';
 import { StoryFn, Parameters } from '@storybook/addons';
 import { DecoratorFunction, LegacyData, StoreData, AddStoryArgs, StoreItem, ErrorLike } from './types';
+import { DebouncedFunc } from 'lodash';
 interface Selection {
     storyId: string;
     viewMode: string;
@@ -47,7 +48,7 @@ export default class StoryStore extends EventEmitter {
         applyDecorators: (fn: StoryFn, decorators: DecoratorFunction[]) => any;
     }): void;
     getStoriesForManager: () => {};
-    pushToManager: (() => void) & import("lodash").Cancelable;
+    pushToManager: DebouncedFunc<() => void>;
     getStoriesForKind(kind: string): StoreItem[];
     getRawStory(kind: string, name: string): StoreItem;
     getRevision(): number;
