diff --git a/node_modules/@types/react-relay/index.d.ts b/node_modules/@types/react-relay/index.d.ts
index 6904123..2e45df1 100644
--- a/node_modules/@types/react-relay/index.d.ts
+++ b/node_modules/@types/react-relay/index.d.ts
@@ -133,7 +133,7 @@ export const ReactRelayContext: React.Context<RelayContext | null>;
 
 export type ContainerProps<Props> = MappedFragmentProps<Pick<Props, Exclude<keyof Props, 'relay'>>>;
 
-export type Container<Props> = React.ComponentType<ContainerProps<Props> & { componentRef?: (ref: any) => void }>;
+export type Container<Props> = React.ComponentType<ContainerProps<Props> & { componentRef?: React.Ref<any> }>;
 
 export function createFragmentContainer<Props>(
     Component: React.ComponentType<Props & { relay?: RelayProp }>,
