diff --git a/node_modules/@types/react-native/index.d.ts b/node_modules/@types/react-native/index.d.ts
index 6322c98..542449f 100644
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -8650,8 +8650,7 @@ export namespace Animated {
 
     export interface AnimatedComponent<
         T extends React.ComponentType<ComponentProps<T>> | React.Component<ComponentProps<T>>
-    > extends React.FC<AnimatedProps<ComponentProps<T>>> {
-        getNode: () => T;
+    > extends React.FC<AnimatedProps<ComponentProps<T>> & {ref?: React.Ref<T | undefined> | null}> {
     }
 
     /**
