diff --git a/node_modules/@types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts b/node_modules/@types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts
index 3dfa8f1..3295579 100755
--- a/node_modules/@types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts
+++ b/node_modules/@types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts
@@ -10,7 +10,7 @@ import {
     NormalizationSplitOperation,
 } from 'relay-runtime';

-interface MockResolverContext {
+export interface MockResolverContext {
     parentType?: string | undefined;
     name?: string | undefined;
     alias?: string | undefined;
diff --git a/node_modules/@types/relay-test-utils/lib/RelayModernMockEnvironment.d.ts b/node_modules/@types/relay-test-utils/lib/RelayModernMockEnvironment.d.ts
index c14ad95..906b2a0 100755
--- a/node_modules/@types/relay-test-utils/lib/RelayModernMockEnvironment.d.ts
+++ b/node_modules/@types/relay-test-utils/lib/RelayModernMockEnvironment.d.ts
@@ -37,9 +37,11 @@ export interface MockFunctions {
     findOperation: (findFn: (operation: OperationDescriptor) => boolean) => OperationDescriptor;
     queuePendingOperation: (query: GraphQLTaggedNode, variables: Variables) => void;
     getMostRecentOperation: () => OperationDescriptor;
+    /** @deprecated Use our `resolveMostRecentRelayOperation` or `resolveMostRecentRelayOperationPayload` helpers instead. */
     resolveMostRecentOperation: (
         payload: GraphQLResponse | ((operation: OperationDescriptor) => GraphQLResponse | void),
     ) => void;
+    /** @deprecated Use our `rejectMostRecentRelayOperation` helper instead. */
     rejectMostRecentOperation: (error: Error | ((operation: OperationDescriptor) => Error)) => void;
     queueOperationResolver: (resolver: OperationMockResolver) => void;
 }
