diff --git a/node_modules/@wora/relay-store/lib/RecordSource.d.ts b/node_modules/@wora/relay-store/lib/RecordSource.d.ts
index 422b7bf..74cfe72 100644
--- a/node_modules/@wora/relay-store/lib/RecordSource.d.ts
+++ b/node_modules/@wora/relay-store/lib/RecordSource.d.ts
@@ -1,5 +1,5 @@
 import { RecordState } from 'relay-runtime';
-import { MutableRecordSource, Record, RecordMap } from 'relay-runtime/lib/store/RelayStoreTypes';
+import { MutableRecordSource, Record } from 'relay-runtime/lib/store/RelayStoreTypes';
 import { DataCache, CacheOptions } from '@wora/cache-persist';
 export interface IMutableRecordSourceOffline extends MutableRecordSource {
     restore(): Promise<DataCache>;
@@ -19,6 +19,6 @@ export declare class RecordSource implements IMutableRecordSourceOffline {
     remove(dataID: string): void;
     set(dataID: string, record: Record): void;
     size(): number;
-    toJSON(): RecordMap;
+    toJSON(): { [key: string]: Record };
 }
 //# sourceMappingURL=RecordSource.d.ts.map
