diff --git a/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m b/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m
index 9f20973..9b09ae4 100644
--- a/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m
+++ b/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m
@@ -124,8 +124,19 @@ - (void) setConfiguration:(NSDictionary *)options
     }
 }
 
-- (UIViewController*) getRootVC {
-    UIViewController *root = [[[[UIApplication sharedApplication] delegate] window] rootViewController];
+- (UIViewController *) getRootVC {
+    UIWindow *presentingWindow;
+    for (UIWindow *window in [[UIApplication sharedApplication] windows]) {
+        if (![window isHidden] && [window windowLevel] == UIWindowLevelNormal) {
+            presentingWindow = window;
+        }
+    }
+
+    if (!presentingWindow) {
+        presentingWindow = [[[UIApplication sharedApplication] windows] firstObject];
+    }
+
+    UIViewController *root = [presentingWindow rootViewController];
     while (root.presentedViewController != nil) {
         root = root.presentedViewController;
     }
