diff --git a/node_modules/react-dom/cjs/react-dom.development.js b/node_modules/react-dom/cjs/react-dom.development.js
index f0b9ee7..f861872 100644
--- a/node_modules/react-dom/cjs/react-dom.development.js
+++ b/node_modules/react-dom/cjs/react-dom.development.js
@@ -1446,7 +1446,9 @@ function getActiveElement(doc) {
   try {
     return doc.activeElement || doc.body;
   } catch (e) {
-    return doc.body;
+    // this is causing occasional flaky failures in our test suite
+    // https://github.com/facebook/react/issues/15691
+    return doc && doc.body;
   }
 }
 
