diff --git a/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm b/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
index de7f368..faeb964 100644
--- a/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
+++ b/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
@@ -468,6 +468,14 @@ namespace react = facebook::react;
   UINavigationItem *navitem = vc.navigationItem;
   UINavigationController *navctr = (UINavigationController *)vc.parentViewController;
 
+  // When modal is shown the underlying RNSScreen isn't attached to any navigation controller.
+  // During the modal dismissal transition this update method is called on this RNSScreen resulting in nil navctr.
+  // After the transition is completed it will be called again and will configure the navigation controller correctly.
+  // Also see: https://github.com/software-mansion/react-native-screens/pull/2336
+  if (navctr == nil) {
+    return;
+  }
+
   NSUInteger currentIndex = [navctr.viewControllers indexOfObject:vc];
   UINavigationItem *prevItem =
       currentIndex > 0 ? [navctr.viewControllers objectAtIndex:currentIndex - 1].navigationItem : nil;
