diff --git a/node_modules/react-native-safe-area-context/jest/mock.tsx b/node_modules/react-native-safe-area-context/jest/mock.tsx
index 81688c0..61a7b99 100644
--- a/node_modules/react-native-safe-area-context/jest/mock.tsx
+++ b/node_modules/react-native-safe-area-context/jest/mock.tsx
@@ -1,5 +1,5 @@
 import { jest } from '@jest/globals';
-import React, { useContext } from 'react';
+import { useContext } from 'react';
 import type { Metrics } from '../src/SafeArea.types';
 import type {
   SafeAreaProviderProps,
