diff --git a/node_modules/react-native-push-notification/component/index.ios.js b/node_modules/react-native-push-notification/component/index.ios.js
deleted file mode 100644
index 82eee6a..0000000
--- a/node_modules/react-native-push-notification/component/index.ios.js
+++ /dev/null
@@ -1,7 +0,0 @@
-"use strict";
-
-import PushNotificationIOS from "@react-native-community/push-notification-ios";
-
-module.exports = {
-  component: PushNotificationIOS
-};
