diff --git a/node_modules/react-native-keys/android/CMakeLists.txt b/node_modules/react-native-keys/android/CMakeLists.txt
index f453363..bbbe697 100644
--- a/node_modules/react-native-keys/android/CMakeLists.txt
+++ b/node_modules/react-native-keys/android/CMakeLists.txt
@@ -9,7 +9,7 @@ set(BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
 if(${REACT_NATIVE_MINOR_VERSION} GREATER_EQUAL 71)
     # Consume shared libraries and headers from prefabs
     find_package(ReactAndroid REQUIRED CONFIG)
-
+    find_package(fbjni REQUIRED CONFIG)  # Add this line to find fbjni
     include_directories(
         ${PACKAGE_NAME}
         "./androidcpp"
@@ -49,7 +49,17 @@ if(${REACT_NATIVE_MINOR_VERSION} GREATER_EQUAL 71)

     find_package(openssl REQUIRED CONFIG)

-    target_link_libraries(
+    if (REACTNATIVE_MERGED_SO OR ReactAndroid_VERSION_MINOR GREATER_EQUAL 76)
+     target_link_libraries(
+        ${PACKAGE_NAME}
+        ${LOG_LIB}
+        ReactAndroid::reactnative
+        ReactAndroid::jsi
+        fbjni::fbjni
+        openssl::crypto
+      )
+    else()
+      target_link_libraries(
         ${PACKAGE_NAME}
         ${LOG_LIB}
         ReactAndroid::jsi
@@ -58,6 +68,7 @@ if(${REACT_NATIVE_MINOR_VERSION} GREATER_EQUAL 71)
         android
         openssl::crypto
     )
+    endif()
 else()
     add_library(
         ${PACKAGE_NAME}
diff --git a/node_modules/react-native-keys/android/build.gradle b/node_modules/react-native-keys/android/build.gradle
index 7d07aae..56f223a 100644
--- a/node_modules/react-native-keys/android/build.gradle
+++ b/node_modules/react-native-keys/android/build.gradle
@@ -156,6 +156,7 @@ android {
       "**/libc++_shared.so",
       "**/libfbjni.so",
       "**/libreactnativejni.so",
+      "**/libreactnative.so",
       "**/libjsi.so",
       "**/libreact_nativemodule_core.so",
       "**/libturbomodulejsijni.so",
