diff --git a/node_modules/react-native-appboy-sdk/react-native-appboy-sdk.podspec b/node_modules/react-native-appboy-sdk/react-native-appboy-sdk.podspec
deleted file mode 100644
index 4e69ac8..0000000
--- a/node_modules/react-native-appboy-sdk/react-native-appboy-sdk.podspec
+++ /dev/null
@@ -1,23 +0,0 @@
-require 'json'
-
-package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
-
-Pod::Spec.new do |s|
-  s.name           = package['name']
-  s.version        = package['version']
-  s.summary        = package['description']
-  s.description    = package['description']
-  s.license        = package['license']
-  s.author         = package['author']
-  s.homepage       = package['homepage']
-  s.source         = { :git => 'git+https://github.com/Appboy/appboy-react-sdk.git', :tag => s.version }
-
-  s.requires_arc   = true
-  s.platform       = :ios, '9.0'
-
-  s.preserve_paths = 'LICENSE.md', 'README.md', 'package.json', 'index.js'
-  s.source_files   = 'iOS/**/*.{h,m}'
-
-  s.dependency 'Appboy-iOS-SDK', '~> 4.2.0'
-  s.dependency 'React'
-end
