diff --git a/node_modules/react-native-sentry/lib/Sentry.d.ts b/node_modules/react-native-sentry/lib/Sentry.d.ts
index 726204e..12d3b5a 100644
--- a/node_modules/react-native-sentry/lib/Sentry.d.ts
+++ b/node_modules/react-native-sentry/lib/Sentry.d.ts
@@ -38,8 +38,6 @@ interface SentryOptions {
   ignoreModulesInclude?: [string];
 }
 
-export default Sentry;
-
 export class Sentry {
   install(): void;
 
