diff --git a/node_modules/react-native-config/react-native-config.podspec b/node_modules/react-native-config/react-native-config.podspec
index 54985dd..e9cab1e 100644
--- a/node_modules/react-native-config/react-native-config.podspec
+++ b/node_modules/react-native-config/react-native-config.podspec
@@ -21,7 +21,16 @@ Pod::Spec.new do |s|
     name: 'Config codegen',
     script: %(
 set -ex
-HOST_PATH="$SRCROOT/../.."
+# Eigen doesn't use standard React Native folder structure (yet) so we have to do this hack.
+HOST_PATH="$SRCROOT/.."
+# Use CI or shared env file.
+if [ -z ${CI+x} ]; then
+  echo "Not running on CI, using shared env file";
+  export ENVFILE=".env.shared";
+else
+  echo "Running on CI, using CI env file";
+  export ENVFILE=".env.ci";
+fi
 "${PODS_TARGET_SRCROOT}/ios/ReactNativeConfig/BuildDotenvConfig.rb" "$HOST_PATH" "${PODS_TARGET_SRCROOT}/ios/ReactNativeConfig"
 ),
     execution_position: :before_compile,
