diff --git a/node_modules/react-native-haptic-feedback/index.d.ts b/node_modules/react-native-haptic-feedback/index.d.ts
index 3c08697..ae08cff 100644
--- a/node_modules/react-native-haptic-feedback/index.d.ts
+++ b/node_modules/react-native-haptic-feedback/index.d.ts
@@ -1,5 +1,5 @@
 declare module "react-native-haptic-feedback" {
-  type HapticFeedbackTypes =
+  export type HapticFeedbackTypes =
     | "selection"
     | "impactLight"
     | "impactMedium"
