diff --git a/node_modules/react-relay-network-modern/node8/middlewares/upload.js b/node_modules/react-relay-network-modern/node8/middlewares/upload.js
index 73aefa7..135c5da 100644
--- a/node_modules/react-relay-network-modern/node8/middlewares/upload.js
+++ b/node_modules/react-relay-network-modern/node8/middlewares/upload.js
@@ -24,7 +24,9 @@ function uploadMiddleware() {
     const {
       clone: extractedOperations,
       files
-    } = (0, _extractFiles.extractFiles)(operations);
+    } = (0, _extractFiles.extractFiles)(operations, "", (value) => {
+      return value instanceof _extractFiles.ReactNativeFile;
+    });
 
     if (files.size) {
       const formData = new FormData();
@@ -37,7 +39,7 @@ function uploadMiddleware() {
       formData.append('map', JSON.stringify(pathMap));
       i = 0;
       files.forEach((paths, file) => {
-        formData.append(++i, file, file.name);
+        formData.append(++i, file);
       });
       req.fetchOpts.method = 'POST';
       req.fetchOpts.body = formData;
