diff --git a/node_modules/react-native-reanimated/lib/reanimated2/globals.d.ts b/node_modules/react-native-reanimated/lib/reanimated2/globals.d.ts
index 4f0c619..a618790 100644
--- a/node_modules/react-native-reanimated/lib/reanimated2/globals.d.ts
+++ b/node_modules/react-native-reanimated/lib/reanimated2/globals.d.ts
@@ -1,3 +1,5 @@
+// @ts-nocheck
+// @ts-nocheck
 import { AnimatedStyle, StyleProps } from './commonTypes';
 import { ReanimatedConsole } from './core';
 import { MeasuredDimensions } from './NativeMethods';
diff --git a/node_modules/react-native-reanimated/react-native-reanimated.d.ts b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
index a681f9f..7b6d1b8 100644
--- a/node_modules/react-native-reanimated/react-native-reanimated.d.ts
+++ b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
@@ -1,3 +1,5 @@
+// @ts-nocheck
+// @ts-nocheck
 // Project: https://github.com/software-mansion/react-native-reanimated
 // TypeScript Version: 2.8
 
diff --git a/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts b/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts
index 0f82055..d6b4439 100644
--- a/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts
+++ b/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts
@@ -16,8 +16,10 @@ export default class JSReanimated extends NativeReanimated {
   constructor() {
     super(false);
     if (isJest()) {
+      // @ts-ignore
       this.timeProvider = { now: () => global.ReanimatedDataMock.now() };
     } else {
+      // @ts-ignore
       this.timeProvider = { now: () => window.performance.now() };
     }
   }
