diff --git a/node_modules/relay-runtime/lib/query/GraphQLTag.js b/node_modules/relay-runtime/lib/query/GraphQLTag.js
index f9e9f32..a28b60d 100644
--- a/node_modules/relay-runtime/lib/query/GraphQLTag.js
+++ b/node_modules/relay-runtime/lib/query/GraphQLTag.js
@@ -27,14 +27,20 @@ function graphql(strings) {
 function getNode(taggedNode) {
   var node = taggedNode;
 
-  if (typeof node === 'function') {
+  // This patch is just to make an error message more helpful. It can be deleted safely.
+  if (typeof taggedNode === 'function') {
     node = node();
-    process.env.NODE_ENV !== "production" ? warning(false, 'RelayGraphQLTag: node `%s` unexpectedly wrapped in a function.', node.kind === 'Fragment' ? node.name : node.operation.name) : void 0;
-  } else if (node["default"]) {
+  }
+
+  if (node["default"]) {
     // Support for languages that work (best) with ES6 modules, such as TypeScript.
     node = node["default"];
   }
 
+  if (typeof taggedNode === 'function') {
+    process.env.NODE_ENV !== "production" ? warning(false, 'RelayGraphQLTag: node `%s` unexpectedly wrapped in a function.', node.kind === 'Fragment' ? node.name : node.operation.name) : void 0;
+  }
+
   return node;
 }
 
