diff --git a/node_modules/sift-react-native/.DS_Store b/node_modules/sift-react-native/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/sift-react-native/lib/typescript/index.d.ts b/node_modules/sift-react-native/lib/typescript/index.d.ts
index e6bcb83..c6f8471 100644
--- a/node_modules/sift-react-native/lib/typescript/index.d.ts
+++ b/node_modules/sift-react-native/lib/typescript/index.d.ts
@@ -1,6 +1,7 @@
 declare type SiftReactNativeType = {
     setSiftConfig(accountId: string, beaconKey: string, disallowCollectingLocationData: boolean, serverUrlFormat: string): void;
     setUserId(userId: string): void;
+    setPageName(pageName: string): void;
     unsetUserId(): void;
     upload(): void;
 };
diff --git a/node_modules/sift-react-native/package.json b/node_modules/sift-react-native/package.json
index 1a2b3c4..c6a8e9e 100644
--- a/node_modules/sift-react-native/package.json
+++ b/node_modules/sift-react-native/package.json
@@ -4,7 +4,7 @@
   "description": "A React Native wrapper for Sift iOS and Android SDKs",
   "main": "lib/commonjs/index",
   "module": "lib/module/index",
-  "types": "lib/typescript/src/index.d.ts",
+  "types": "lib/typescript/index.d.ts",
   "react-native": "src/index",
   "source": "src/index",
   "files": [
