diff --git a/node_modules/sift-react-native/lib/typescript/src/index.d.ts b/node_modules/sift-react-native/lib/typescript/src/index.d.ts
index e6bcb83..6582ec8 100644
--- a/node_modules/sift-react-native/lib/typescript/src/index.d.ts
+++ b/node_modules/sift-react-native/lib/typescript/src/index.d.ts
@@ -1,6 +1,7 @@
 declare type SiftReactNativeType = {
     setSiftConfig(accountId: string, beaconKey: string, disallowCollectingLocationData: boolean, serverUrlFormat: string): void;
     setUserId(userId: string): void;
+    setPageName(pageName: String): void;
     unsetUserId(): void;
     upload(): void;
 };
