diff --git a/node_modules/whatwg-fetch/dist/fetch.umd.js b/node_modules/whatwg-fetch/dist/fetch.umd.js
index f9b44fd..331f863 100644
--- a/node_modules/whatwg-fetch/dist/fetch.umd.js
+++ b/node_modules/whatwg-fetch/dist/fetch.umd.js
@@ -444,6 +444,28 @@
     exports.DOMException.prototype.constructor = exports.DOMException;
   }

   function fetch(input, init) {
     return new Promise(function(resolve, reject) {
       var request = new Request(input, init);
@@ -452,6 +474,43 @@
         return reject(new exports.DOMException('Aborted', 'AbortError'))
       }

       var xhr = new XMLHttpRequest();
       function abortXhr() {
