patch-package
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -8435,8 +8435,9 @@ export type Clipboard = ClipboardStatic;
 export var DatePickerAndroid: DatePickerAndroidStatic;
 export type DatePickerAndroid = DatePickerAndroidStatic;
 
-export var Geolocation: GeolocationStatic;
-export type Geolocation = GeolocationStatic;
+// https://github.com/DefinitelyTyped/DefinitelyTyped/issues/24573
+// export var Geolocation: GeolocationStatic;
+// export type Geolocation = GeolocationStatic;
 
 /** http://facebook.github.io/react-native/blog/2016/08/19/right-to-left-support-for-react-native-apps.html */
 export var I18nManager: I18nManagerStatic;
