patch-package
--- a/node_modules/@react-native-community/cameraroll/typings/CameraRoll.d.ts
+++ b/node_modules/@react-native-community/cameraroll/typings/CameraRoll.d.ts
@@ -86,6 +86,6 @@ export interface CameraRollStatic {
   getPhotos: (params: GetPhotosParams) => Promise<PhotoIdentifiersPage>;
 }
 
-let CameraRoll: CameraRollStatic;
+declare let CameraRoll: CameraRollStatic;
 
 export default CameraRoll;
